### DISCUSSION TITLE
Energy Usage Dashboard - Showing energy collected today as todays usage, when it should be yesterday

### DISCUSSION THREAD

**Discussion by wilki2208** [BottlecapDave/HomeAssistant-OctopusEnergy#652] [Q&A] [answered]
Hi,

I have this integration enabled and also added the entity of "sensor.octopus_energy_electricity_21e5091217_XXXXXX_previous_accumulative_consumption" to my Energy dashboard, it seems to not bring in the usage on a per hour basis and instead just drops the previous days energy usage into the today section and as an entry of 1 hour, rather than spread across.

I've checked the entity and in the attributes it shows the below, example:
`charges: 
- start: '2023-12-20T00:00:00+00:00'
  end: '2023-12-20T00:30:00+00:00'
  consumption: 1.359
- start: '2023-12-20T00:30:00+00:00'
  end: '2023-12-20T01:00:00+00:00'
  consumption: 1.327`

I can only imagine I have this setup wrong.  Any advice would be great.

**Comment by BottlecapDave** [comment, marked as answer]
Hello. This is unfortunately a limitation with the HA entities. Please see the [faq](https://bottlecapdave.github.io/HomeAssistant-OctopusEnergy/faq/#ive-added-my-previous-consumption-sensors-to-the-energy-dashboard-but-they-are-reported-in-a-single-chunk-and-are-a-day-or-more-out-is-this-a-bug) for more information
